src/lib/ui/ExtensionByCategoryCard.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+import type { CatalogExtensionInfo } from '../api/extensions-info';
+import { setCurrentExtension } from '../extensions.svelte';
+import ExtensionIcon from './ExtensionIcon.svelte';
+
+const { extension }: { extension: CatalogExtensionInfo } = $props();
+
+const latestVersion = $derived.by(() => {
+  const version = extension.versions?.[0]?.version ?? '';
+  // add a "v" in front of the version number if it's not there
+  return version.startsWith('v') ? version : `v${version}`;
+});
+
+function hideDetails(): void {
+  setCurrentExtension(extension);
+}
+</script>
+
+<button
```

Comment:
Button componenrt ?

Reply:
so here it's using button to say that we have an active element with a onclick trigger (a card) but it doesn't look at all as a button